libexec/python/sutils.py

```diff
@@ -528,3 +530,32 @@ def write_singularity_infos(base_dir,
                                  extension)
     write_file(output_file, content)
     return output_file
+
+
+def get_singularity_conf_value(key):
```

Comment:
Is the absolute only way to do this to read in the actual config file and then search every line for the value of interest? My intuition is that as long as there aren't many values, and primary interaction of the python client with everything else is via the environment, this should be more controlled (e.g., if a value is needed from the config it should be exported in the environment, defined in defaults.py, and then imported from there.

Reply:
Yeah, maybe. I did it this way as I didn't at the time want to touch the C code and stick more stuff out into the environment. I'm a little confused what the preferred thing is to do to get config settings in different parts of the code base? Are the C libexec binaries always taking from environment, or sometimes reading from conf file? Wasn't there some discussion of this on the C side in another PR where .@GodloveD  and you had both implemented something to read config - might be going crazy and imagining though.